spec/cypress/integration/a11y.js

```diff
@@ -36,7 +36,11 @@ describe("Accessibility", () => {
     cy.checkA11y();
 
     cy.get('[action="/users/sign_in_with_token"] [name="commit"]').click();
-    cy.get("h1").should("contain", "User dashboard");
+    cy.get("@userData").then((user) => {
+      // Update user as previous step caused login token to change
```

Comment:
you weren't messing around when you were talking about copying and pasting test code 🤪

Reply:
ah crap